README.md

```diff
@@ -0,0 +1,24 @@
+### scalagen
+[![Join the chat at https://gitter.im/scalameta/scalameta](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/scalameta/scalameta)
+
+Fast scalameta powered code generation.
+
+### User documentation
+Head over to [scalameta user docs][docs] to learn more about the project and its roadmap.
+
+### Tutorial
+If you'd like to find out how to use scalameta, see this [tutorial](http://scalameta.org/tutorial).
+
+### Team
+The current maintainers (people who can merge pull requests) are:
+
+* Eugene Burmako - [`@xeno-by`](https://github.com/xeno-by)
```

Comment:
I propose we keep this

* Ólafur Páll Geirsson - [`@olafurpg`](https://github.com/olafurpg)
* David Dudson - [`@DavidDudson`](https://github.com/DavidDudson)

for now. The list should reflect the people actively working on the project.

Reply:
:+1: